src/main/java/de/flo56958/minetinker/modifiers/ModManager.java

```diff
@@ -240,6 +240,16 @@ private ModManager() {}
 		};
 	}
 
+	public List<Modifier> getToolMods(ItemStack tool) {
+		ArrayList<Modifier> mods = new ArrayList<>();
+		for (Modifier mod : getAllMods()) {

```

Comment:
Careful! getAllMods() returns all Modifiers that the Plugin knows. Some of which could be disabled. You should use getAllowedMods() instead.

Reply:
I was originally going to correct that but seems like I forgot to do so before pushing